include/FastNoise/FastNoise_BuildList.inl

```diff
@@ -90,6 +90,7 @@ FASTSIMD_BUILD_CLASS( FASTNOISE_CLASS( Value ) )
 FASTSIMD_BUILD_CLASS( FASTNOISE_CLASS( Perlin ) )
 FASTSIMD_BUILD_CLASS( FASTNOISE_CLASS( Simplex ) )
 FASTSIMD_BUILD_CLASS( FASTNOISE_CLASS( OpenSimplex2 ) )
+FASTSIMD_BUILD_CLASS( FASTNOISE_CLASS( OpenSimplex2S ) )
```

Comment:
Add this to the bottom of the list so as not to break existing encoded nodes trees